crates/solvers/src/domain/solver/baseline.rs

```diff
@@ -153,9 +159,9 @@ impl Inner {
                     output.amount = cmp::min(output.amount, order.buy.amount);
                 }
 
+                let gas = route.gas() + self.solution_gas_offset;
```

Comment:
We already add the settlement gas overhead inside `Single::IntoSettlement` when computing the actual fee:

https://github.com/cowprotocol/services/blob/ac6580a2870099afc150c8e1527f1977dd659c74/crates/solvers/src/domain/solution.rs#L170-L177

Wouldn't this be the better place to add this? Maybe we could replace the settlement overhead with this parameter and increase it by whatever value @harisang suggested.

Reply:
Yeah, good idea. Conceptually the gas overhead should be a constant (as it should not depend on the interactions we do) so I didn't think about touching that. But that constant is obviously not good enough to make all estimators accurate so we might as well ditch it completely and only have a single configurable gas offset (possibly with the current constant value as the default).